homeassistant/components/volumio/media_player.py

```diff
@@ -3,6 +3,8 @@
 
 Volumio rest API: https://volumio.github.io/docs/API/REST_API.html
 """
+from __future__ import annotations
+
 from datetime import timedelta
```

Comment:
`__future__` is no longer required since removing `_CONFIGURING`:
```suggestion
from datetime import timedelta
```